addon/components/publishr/draft-display/article/template.hbs

```diff
@@ -1,7 +1 @@
-{{#if editable}}
-  {{upf-editor height=500 btnSize=bs-sm content=(readonly item.body)
-               onContentChange=(action 'onContentChange')
-               uploaderHeaders=uploaderHeaders uploaderExtra=uploaderExtra}}
-{{else}}
```

Comment:
`editable` was never passed to the article component anywhere. If at some point we need it, it will be done using UEdit this time.